app/domain/operations/fdsh/ridp/request_secondary_determination.rb

```diff
@@ -58,8 +58,9 @@ def create_payload_entity(value)
           Success(AcaEntities::Families::Family.new(value.to_h))
         end
 
-        def publish(payload)
-          Operations::Fdsh::Ridp::PublishSecondaryRequest.new.call(payload)
+        def publish(payload, interactive_verification)
+          session_id = interactive_verification&.session_id ? interactive_verification.session_id : nil
```

Comment:
very minor but `interactive_verification&.session_id` will equal nil, so no need for the ternary here, we can just do the `interactive_verification&.session_id`